dnf-docker-test/features/steps/test_behave.py

```diff
@@ -228,3 +229,20 @@ def then_file_contein(context, path_to_file, content):
     assert os.path.isfile(path[0]), "The file {} is not a file or doesn't exist".format(path[0])
     with open(path[0], 'r') as f:
         assert content in f, "The file {} doesn't contain '{}'".format(path[0], content)
+
+
+@then('the "{section}" section should contain package "{pkg}"')
+def then_the_section(context, section, pkg):
+    lines = iter(context.cmd_output.split('\n'))
+    try:
+        while not re.match('^' + section + ':', lines.next()):
```

Comment:
What about to test not only single line below section name, but all that start with '  ' ? Then it can search for multiple packages in that section.